PropertyTest1.fs

```diff
@@ -0,0 +1,140 @@
+module Lab2.PropertyTest1
+
+open FsCheck
+open FsCheck.NUnit
+open Lab2
+
+let rec isGraphValid graph =
```

Comment:
I guess, you can skip most of type hints in that file as compiler will determine type by isGraphValid

Reply:
Or maybe you can write own generator to avoid use of isGraphValid in tests

Reply:
That would be better actually